backend/.jenkins/common.sh

```diff
@@ -2,6 +2,8 @@
 
 set -ex
 
+export OS2MO_DUMMY_MODE=true
```

Comment:
You shouldn't need to set that here; the tests should assign it themselves as needed. After all, developers frequently run the tests outside Jenkins 🙂

Reply:
Yes - it is a remnant. Sorry. It is set in tests/util.py also.

Reply:
> Yes - it is a remnant. Sorry. It is set in tests/util.py also.

Could you please remove it, then? 🙂

Reply:
done - will be exciting to see how Jenkins copes.